framework/platform/bpl/CMakeLists.txt

```diff
@@ -8,6 +8,23 @@ set(MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 # Common Sources
 file(GLOB_RECURSE bpl_common_sources ${MODULE_PATH}/common/*.c*)
 
+# Common prplmesh_db values
+set(BEEROCKS_MANAGEMENT_MODE  "Multi-AP-Controller-and-Agent" CACHE STRING "Set management mode")
+set(BEEROCKS_OPERATING_MODE  "Gateway" CACHE STRING "Set operating mode")
+set(BEEROCKS_STOP_ON_FAILURE_ATTEMPTS  0 CACHE INT "In common config stop on failure disabled")
+set(BEEROCKS_CERTIFICATION_MODE  1 CACHE INT "In certification, agent must retry onboarding indefinitely")
+
+set(BEEROCKS_HOSTAP_WLAN1_CTRL_IFACE "/var/run/hostapd/${BEEROCKS_WLAN1_IFACE}" CACHE PATH "hostapd ctrl iface path for 1st WLAN")
+set(BEEROCKS_HOSTAP_WLAN2_CTRL_IFACE "/var/run/hostapd/${BEEROCKS_WLAN2_IFACE}" CACHE PATH "hostapd ctrl iface path for 2nd WLAN")
+set(BEEROCKS_HOSTAP_WLAN3_CTRL_IFACE "/var/run/hostapd/${BEEROCKS_WLAN3_IFACE}" CACHE PATH "hostapd ctrl iface path for 3rd WLAN")
+
+set(BEEROCKS_WPA_SUPPLICANT_WLAN1_CTRL_IFACE "/var/run/wpa_supplicant/${BEEROCKS_WLAN1_IFACE}" CACHE PATH "wpa_supplicant ctrl iface path for 1st WLAN")
+set(BEEROCKS_WPA_SUPPLICANT_WLAN2_CTRL_IFACE "/var/run/wpa_supplicant/${BEEROCKS_WLAN2_IFACE}" CACHE PATH "wpa_supplicant ctrl iface path for 2nd WLAN")
+set(BEEROCKS_WPA_SUPPLICANT_WLAN3_CTRL_IFACE "/var/run/wpa_supplicant/${BEEROCKS_WLAN3_IFACE}" CACHE PATH "wpa_supplicant ctrl iface path for 3rd WLAN")
+
+# Add prplmesh_platform_db config generation
+add_subdirectory(platform_db)
```

Comment:
This should only be done for the Linux implementation, not for UCI.

Reply:
I disagree with you @arnout! We need a prplmesh_platform_db file in all platforms. In the current master, it generates by prplmesh_utils.sh and it's weird. I think we should generate prplmesh_platform_db  by CMake for all platforms.